header.go

```diff
@@ -1853,11 +1858,17 @@ func (h *ResponseHeader) peekAll(key []byte) [][]byte {
 	h.mulHeader = h.mulHeader[:0]
 	switch string(key) {
 	case HeaderContentType:
-		h.mulHeader = append(h.mulHeader, h.ContentType())
+		if contentType := h.ContentType(); len(contentType) > 0 {
```

Comment:
This is a modification from VisitAll, because I found that when using PeekAll, if the field is empty he returns `[][]byte{ '' }` instead of `[][]byte{}`.